uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -456,6 +472,13 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
         return resources.getString(R.string.karhoo_uisdk_locale)
     }
 
+    override fun isTermsCheckBoxValid(): Boolean {
+        return when(KarhooUISDKConfigurationProvider.configuration.useCheckboxOnTermsAndConditions()){
+            true -> bookingRequestTermsWidget.khTermsAndConditionsCheckBox.isChecked
+            false -> true
```

Comment:
a bit weirdish the syntax, might be simpler with an if

if(KarhooUISDKConfigurationProvider.configuration.useCheckboxOnTermsAndConditions()) return bookingRequestTermsWidget.khTermsAndConditionsCheckBox.isChecked

return true